app/javascript/tests/close_contact/CloseContact.test.js

```diff
@@ -265,49 +265,94 @@ describe('CloseContact', () => {
   });
 
   it('Properly enables and disables the submit/create button when the correct fields are entered or removed', () => {
```

Comment:
Just a thought, maybe break each combination into a separate test?  I also think it's fine but I would remove the comment

Reply:
Agreed ideally these would be separated out, much easier to find the issue in those cases.